typing_tests/utils.ts

```diff
@@ -1,4 +1,5 @@
-import { Any, Boolean } from 'ts-toolbelt';
+import { Any, Boolean, } from 'ts-toolbelt';
```

Comment:
To keep it consistent we can remove the `,` as long as it's on the same line

Reply:
That's weird and probably means that we do not run linting as a test on ci...